lib/fill/index.js

```diff
@@ -0,0 +1,13 @@
+import curryN from '../curryN'
+import toArray from '../toArray'
+
+export default curryN(2, function fill(val, start, end, arr) {
+  var last = arguments[arguments.length - 1]
+  return Array.isArray(last)
+    ? val
+      ? toArray(last).fill(val, Array.isArray(start) ? undefined : start, Array.isArray(end) ? undefined : end)
```

Comment:
no, no and again no
![image](https://user-images.githubusercontent.com/2991847/35450044-e0e6d722-02cf-11e8-8d60-f8fcd0274f36.png)